server/routes/addPatient/addPatientCompleted.test.ts

```diff
@@ -2,32 +2,33 @@ import { Express } from 'express'
 import request from 'supertest'
 import { appWithAllRoutes, mockJwtDecode } from '../testutils/appSetup'
 import PrisonerSearchService, { PrisonerResultSummary } from '../../services/prisonerSearchService'
-import HospitalSearchService, { Hospital } from '../../services/hospitalSearchService'
+import AgencySearchService from '../../services/agencySearchService'
+import { Prison } from '../../data/prisonApiClient'
 
 jest.mock('../../services/prisonerSearchService')
-jest.mock('../../services/hospitalSearchService')
+jest.mock('../../services/agencySearchService')
 
 const prisonerSearchService = new PrisonerSearchService(null) as jest.Mocked<PrisonerSearchService>
-const hospitalSearchService = new HospitalSearchService() as jest.Mocked<HospitalSearchService>
+const agencySearchService = new AgencySearchService() as jest.Mocked<AgencySearchService>
 
 let app: Express
 
 describe('GET /prisoner-added', () => {
   beforeEach(() => {
     app = appWithAllRoutes({
       production: false,
-      services: { prisonerSearchService, hospitalSearchService },
+      services: { prisonerSearchService, agencySearchService },
       session: { newAddRestrictedPatientJourney: true },
       roles: ['RESTRICTED_PATIENT_MIGRATION'],
     })
 
-    hospitalSearchService.getHospital.mockResolvedValue({
+    agencySearchService.getAgency.mockResolvedValue({
       agencyId: 'SHEFF',
       description: 'Sheffield Hospital',
       longDescription: 'Sheffield Teaching Hospital',
       agencyType: 'HOSP',
       active: true,
-    } as Hospital)
```

Comment:
this is an odd one - the service has always returned a `Prison`

Reply:
although I guess it should actually be an `Agency`, will do in next PR...

Reply:
Not sure I get it - wouldn't agencyType by INST if this was returning a prison?